ui/src/components/MyPollsPreview/MyPollsPreview.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from 'react-dom';
+import './MyPollsPreview.css';
+
+class MyPollsPreview extends React.Component {
```

Comment:
How come this specific component is a class component? Every other component here is functional, as well as most in the codebase. We'd best rewrite this to a functional component. Considering its simplicity, there aren't that many changes.